core/common/src/main/resources/alluxio-default.properties

```diff
@@ -107,6 +107,7 @@ alluxio.worker.data.server.class=alluxio.worker.netty.NettyDataServer
 alluxio.worker.evictor.class=alluxio.worker.block.evictor.LRUEvictor
 alluxio.worker.evictor.lrfu.attenuation.factor=2.0
 alluxio.worker.evictor.lrfu.step.factor=0.25
+alluxio.worker.evictor.lirs.hir.percent=0.25
```

Comment:
Please add this to [the documentation](https://github.com/Alluxio/alluxio/blob/master/docs/_data/table/en/worker-configuration.yml) and [the chinese version](https://github.com/Alluxio/alluxio/blob/master/docs/_data/table/cn/worker-configuration.yml).